docs/DeveloperGuide.adoc

```diff
@@ -831,35 +835,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |new JCRC member |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |JCRC member |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |JCRC member |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |JCRC member |list all persons |
```

Comment:
Not sure if it matters, but the plural for person is people.

Reply:
I followed the words of the original developer guide, should be fine